src/lib/browser/BrowserPage.ts

```
@@ -668,6 +670,224 @@ export class BrowserPage {
+  }
+
+  // Use CDP via chrome.debugger to retrieve resource bytes to bypass CORS
+  private async _fetchPdfBytesViaDebugger(url: string): Promise<Uint8Array> {
+    return new Promise<Uint8Array>((resolve, reject) => {
+      const target = { tabId: this._tabId } as chrome.debugger.Debuggee
+      const version = '1.3'
+      let attached = false
+
+      const detach = () => {
+        if (attached) {
+          try { chrome.debugger.detach(target, () => {}) } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)